src/bots/depositor.py

```diff
@@ -182,25 +139,15 @@ def _deposit_to_module(self, module_id: int) -> bool:
         gas_is_ok = self._gas_price_calculator.is_gas_price_ok(module_id)
         logger.info({'msg': 'Calculate gas recommendations.', 'value': gas_is_ok})
 
-        strategy, is_mellow = self._select_strategy(module_id)
-        is_deposit_amount_ok = self._gas_price_calculator.calculate_deposit_recommendation(strategy, module_id)
-        logger.info({'msg': 'Calculations deposit recommendations.', 'value': is_deposit_amount_ok})
-
-        if is_depositable and quorum and can_deposit and gas_is_ok and is_deposit_amount_ok:
-            logger.info({'msg': 'Checks passed. Prepare deposit tx.', 'is_mellow': is_mellow})
-            success = self.prepare_and_send_tx(quorum, is_mellow, self._flashbots_works)
```

Comment:
Maybe better was just adding here smth like
```
success = self.prepare_and_send_tx(quorum, is_mellow, self._flashbots_works)

if not success and is_mellow:
    self.prepare_and_send_tx(quorum, False, self._flashbots_works)
```

And remove `_mellow_works` from class Depositor?

Reply:
Done